### PR TITLE
Update + Bugfix

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: src/Forms/FixCommonErrors.cs [hunk 1]
```diff
@@ -2191,7 +2191,7 @@ public static string FixStartWithUppercaseLetterAfterParagraph(Paragraph p, Para
                 if (prev != null)
                     prevText = HtmlUtil.RemoveHtmlTags(prev.Text);
 
-                bool isPrevEndOfLine = FixCommonErrorsHelper.IsPrevoiusTextEndOfParagraph(prevText);
+                bool isPrevEndOfLine = FixCommonErrorsHelper.IsPreviousTextEndOfParagraph(prevText);
```
Comment (ivandrofly) [inline src/Forms/FixCommonErrors.cs]: Typos

File: src/Forms/ChangeCasing.cs [hunk 2]
```diff
@@ -95,7 +95,7 @@ internal void FixCasing(Subtitle subtitle, string language)
                                 prev = p.Text[match.Index - 1].ToString(CultureInfo.InvariantCulture);
                             if (match.Index + 1 < p.Text.Length)
                                 next = p.Text[match.Index + 1].ToString(CultureInfo.InvariantCulture);
-                            if (prev != ">" && next != ">")
+                            if (prev != "<" || next != ">")
```
Comment (ivandrofly) [inline src/Forms/ChangeCasing.cs]: Bug?! /cc @niksedk

File: src/Logic/StripableText.cs [hunk 3]
```diff
@@ -258,7 +258,7 @@ public void FixCasing(List<string> namesEtc, bool changeNameCases, bool makeUppe
                         {
                             if (s == ']' && sb.ToString().IndexOf('[') > 1)
                             { // I [Motor roaring] love you!
-                                string temp = sb.ToString().Substring(0, sb.ToString().IndexOf('[') - 1).Trim();
```
Comment (ivandrofly) [inline src/Logic/StripableText.cs]: StringBuilder's .ToString() behaved like Substring()

File: src/Forms/ChangeCasing.cs (hunk 2 above)
Comment (niksedk) [inline src/Forms/ChangeCasing.cs]: Well, seems a bit strange... I've tested somewhat and that regular expression is not the best for cases like "but<i>i was<i> bad" - I've tried to fix it here: c2a3d33f23a9f977e5283951e25399adb2eab3c9

**Comment by ivandrofly** [comment]
Okay, I'll try to rebase this branch then

**Pull request closed without merging**